_posts/guides/2019-08-23-guide_install-css-assets.md

```diff
@@ -7,4 +7,42 @@ tags:
   - cs:s
 ---
 
-TODO: Show how to install CS:S assets from SteamCMD
+
+This guide covers how to install CS:S assets using SteamCMD.
+## Prerequisites 
+- An internet connection.
+- Some free space on your PC. (~4GB)
+
+## Getting SteamCMD
+For Linux/Mac refer to Valve's guide:
+https://developer.valvesoftware.com/wiki/SteamCMD
+### Windows:
+- Make a folder for SteamCMD.
+- Inside of it download [***SteamCMD***](https://steamcdn-a.akamaihd.net/client/installer/steamcmd.zip), and unzip it. 
+- Start steamcmd.exe and wait until you got 'Steam>' waiting for your input.
+
+## Getting CSS files
+
+Inside SteamCMD, type or paste each of the following line in the order:
+- `login anonymous`
+- `force_install_dir ./css`
+- `app_update 232330 validate`
+
+## Installing CSS files
+
+- Inside your css folder move the folder `cstrike` at the root of the momentum folder.
+- Inside the momentum folder open `momentum\gameinfo.txt`:
+  - Find:
+  ```
+  game                "C:\Program Files (x86)\Steam\steamapps\common\Counter-Strike Source\cstrike\cstrike_pak.vpk"
+  game                "C:\Program Files (x86)\Steam\steamapps\common\Counter-Strike Source\cstrike\download"
+  game                "C:\Program Files (x86)\Steam\steamapps\common\Counter-Strike Source\cstrike"
+  ```
```

Comment:
Most players are not going to have this in their gameinfo, I'm guessing you did originally to get it to manually mount CS:S stuff.

Reply:
Those lines are present by default in the gameinfo.txt from 0.8.7-Public-Win.